### DISCUSSION TITLE
broker：Too many requests to the same Bookie while reading

### DISCUSSION THREAD

**Discussion by xiaozhuang-a** [milvus-io/milvus#26092] [Q&A and General discussion]
```bash
02:02:04.475 [BookKeeperClientWorker-OrderedExecutor-15-0] INFO  org.apache.bookkeeper.client.PendingReadOp - Error: Too many requests to the same Bookie while reading L3380751 E24554 from bookie: milvus-pulsar-bookie-4.milvus-pulsar-bookie.prod-milvus.svc.cluster.local:3181

```

pulsar broker，I have been reporting the above error and would like to ask how to troubleshoot and handle it

**Comment by yhmo** [comment]
We didn't see this error before.
The pulsar service is deployed by yourself or by the milvus-helm?

**Reply by xiaozhuang-a** [reply]
milvus-helm

**Reply by yhmo** [reply]
It might be an unknown issue. If you encountered it again, please collect the logs of milvus and pulsar, and create an issue for us.
